lib/testing/src/lib/core/pages/form/form-fields.ts

```diff
@@ -124,9 +125,9 @@ export class FormFields {
     }
 
     async selectForm(formName): Promise<void> {
-        await BrowserActions.click(this.selectFormDropDownArrow);
-        await BrowserVisibility.waitUntilElementIsVisible(this.selectFormContent);
-        await this.selectFormFromDropDown(formName);
+        await this.selectFormDropdown.clickDropdown();
+        await this.selectFormDropdown.checkOptionsPanelIsDisplayed();
+        await this.selectFormDropdown.selectOption(formName);
```

Comment:
Let’s try to reduce the methods that need to be called. Can I select an option in the dropdown without clicking on it?
```await clickDropdown();
await selectOption(selectItem);```
If now let’s delegate the click to the selectOption itself with a method
```clickDropdownWithOption(option)```